greek/offer_marshmallow_a_cigar/offer_marshmallow_a_cigar.md

```diff
@@ -1,4 +1,10 @@
 Βγάζεις το πακέτο σου και προσφέρεις στο σαστισμένο χέρι τσιγάρο.
 Τεντώνει όλα τα δάχτυλα του και καταλαβαίνεις ότι έχει εκπλαγεί. Γελάς
 δυνατά και του λες περιπαικτικά ότι ακόμη και σ'ένα δωμάτιο από ζαχαρωτά,
-υπάρχει ένας αγενής Έλληνας καπνιστής.'
\ No newline at end of file
+υπάρχει ένας αγενής Έλληνας καπνιστής.'
+
+Αφου καπνίσεις το τσιγάρο που το ζαχαρωτό χέρι αρνήθηκε σκέφτεσαι
+πως είναι ώρα να βγεις από εκεί. Πόρτα δεν υπάρχει οπότε πρέπει
+να χρησιμοποιήσεις κάτι από αυτά που έχεις. Τι κάνεις;
+
+[Ανάβεις τον αναπτήρα](set_it_on_fire/set-it-on-fire.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.